src/website/views/components/alert/_examples.pug

```diff
@@ -552,6 +552,94 @@ p.-text
           </button>
         </div>
 
+h4 Closable alert interaction
+p.-text
+.example.-mb--3
+  .-p--3#interactive-wrapper
+    chi-button#alert-trigger-button(color='primary')
+      | Click me to show the alert
+    chi-alert.-mt--2#interactive-alert(title='Warning', color='warning', icon='warning', closable)
+      | This is a closable warning alert
+  .example-tabs.-pl--2
+    ul.chi-tabs#example-interactive
+      li.-active
+        a(href=`#webcomponent-portal-interactive`) Web Component
+      li
+        a(href=`#html-portal-interactive`) HTML Blueprint
+  .chi-tabs-panel.-active#webcomponent-portal-interactive
+    .chi-tab__description
+      span.-text--grey
+        | This HTML example requires additional JavaScript.
+    :code(lang="html")
+      <!-- Wrapper -->
+      <div id="interactive-wrapper">
+        <!-- Trigger -->
+        <chi-button id="alert-trigger-button" color="primary">Click me to show the alert</chi-button>
+
+        <!-- Alert -->
+        <chi-alert id="interactive-alert" icon="circle-info" closable mutable>
+          This is a closable warning alert
+        </chi-alert>
```

Comment:
@mtadejalumen code snippets are not refelecting warning alert in web component and html blueprint. Please update